ServerlessLibraryAPI/CosmosLibraryStore.cs

```diff
@@ -66,12 +66,11 @@ public static async Task<T> GetItemAsync(string id)
             }
         }
 
-        public static async Task<List<T>> GetItemsAsync(Expression<Func<T, bool>> predicate)
+        public static async Task<List<T>> GetItemsAsync()
```

Comment:
This API was meant to handle fetching items with predicated. Now this fetches all items. If you feel this generic handling is not required, then change the method name to GetAllItemsAsync

Reply:
I removed the predicate since we don't need it as of now. We can add it back as needed. Renamed as per suggestion